src/euphorie/client/browser/templates/webhelpers.pt

```diff
@@ -368,250 +369,256 @@
+                </label>
+                <label>
+                  <input checked="${python:'checked' if value==7 else None}"
+                         name="frequency:int"
+                         type="radio"
+                         value="7"
+                  />
+                  <tal:span i18n:translate="frequency_french_often">Often</tal:span><br />
+                  <dfn class="icon info pat-tooltip"
+                       title="Once per week"
+                       i18n:attributes="title frequency_french_often_help;"
+                  ></dfn>
+                </label>
+                <label>
+                  <input checked="${python:'checked' if value==9 else None}"
+                         name="frequency:int"
+                         type="radio"
+                         value="9"
+                  />
+                  <tal:span i18n:translate="frequency_french_regularly">Very often or regularly</tal:span><br />
+                  <dfn class="icon info pat-tooltip"
+                       title="Minimum once per day"
+                       i18n:attributes="title frequency_french_regularly_help;"
+                  ></dfn>
+                </label>
+              </fieldset>
             </fieldset>
           </tal:french>
         </tal:get>
       </tal:block>
 
       <tal:block condition="python:evaluation_method=='direct'">
-        <fieldset class="pat-checklist radio"
-                  disabled="${python:'disabled' if not webhelpers.can_edit_session else None}"
-                  tal:define="
-                    value context/priority;
-                  "
-        >
-          <p i18n:translate="header_risk_priority">Select the priority of this risk</p>
-          <label><input checked="${python:'checked' if value=='low' else None}"
-                   name="priority"
-                   type="radio"
-                   value="low"
-            />
-            <tal:span i18n:translate="priority_low">Low</tal:span></label>
-          <label><input checked="${python:'checked' if value=='medium' else None}"
-                   name="priority"
-                   type="radio"
-                   value="medium"
-            />
-            <tal:span i18n:translate="priority_medium">Medium</tal:span></label>
-          <label><input checked="${python:'checked' if value=='high' else None}"
-                   name="priority"
-                   type="radio"
-                   value="high"
-            />
-            <tal:span i18n:translate="priority_high">High</tal:span></label>
+        <fieldset class="vertical">
+          <fieldset class="pat-checklist"
+                    disabled="${python:'disabled' if not webhelpers.can_edit_session else None}"
+                    tal:define="
+                      value context/priority;
+                    "
+          >
+            <legend><tal:span i18n:translate="header_risk_priority">Select the priority of this risk</tal:span></legend>
```

Comment:
```suggestion
            <legend i18n:translate="header_risk_priority">Select the priority of this risk</legend>
```

Reply:
Why the extra `<tal:span>`?

Reply:
Yes, I just replicated the markup from the sections above where there was more in the legend than just text and therefore a tal:span was used. I am fine if you want to change that.